.github/workflows/unstale.yml

```
@@ -0,0 +1,15 @@
+name: Remove Labels
+
+on: [issue_comment]
+
+jobs:
+  remove_labels:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions-ecosystem/action-remove-labels@v1
+        if: ${{ github.event.comment.user.url != 'https://github.com/apps/github-actions' }}
+        with:
+          github_token: ${{ secrets.github_token }}
```

I guess it is case insensitive, but can we use secrets.GITHUB_TOKEN instead, as we do in the rest of the workflows?

---

Good point, it was copy pasted

```suggestion
          github_token: ${{ secrets.GITHUB_TOKEN }}
```